static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -115,27 +109,12 @@ define([
             // need to explicitly set the dimensions for the ima plugin.
             player.height(bonzo(player.el()).parent().dim().height);
             player.width(bonzo(player.el()).parent().dim().width);
-
```

Comment:
This basically said - if html5 fails, reverse the techOrder (`['flash', 'html5']`) and try again.